src/lib/sedlexing.mli

```diff
@@ -86,6 +94,14 @@ val lexeme_length: lexbuf -> int
         lexbuf)], that is, the length (in code points) of the matched
         string. *)
 
+val lexing_positions : lexbuf -> Lexing.position*Lexing.position
+    (** [Sedlexing.lexing_positions lexbuf] returns the start and current
+        positions with type [Lexing.position], suitable for consumption
+        by parsers like [Menhir]. *)
+
+val new_line: lexbuf -> unit
+    (** [Sedlexing.new_line lexbuf] tolds lexbuf that  *)
+
```

Comment:
that ? :)

Reply:
Sorry, I must forget finishing the document at that time (it's tiresome writing document, isn't it :))